README.md

```diff
@@ -242,6 +242,46 @@ evaluation.run(model)
 
 </details>
 
+<details>
+  <summary>  Using a cross encoder for reranking</summary>
+
+
+### Using a cross encoder for reranking
+
+To use a cross encoder for reranking, you can directly use a CrossEncoder from SentenceTransformers. The following code shows a two-stage run with the second stage reading results saved from the first stage. 
+
+```python
+from mteb import MTEB
+from sentence_transformers import CrossEncoder, SentenceTransformer
+
+cross_encoder = CrossEncoder("cross-encoder/ms-marco-TinyBERT-L-2-v2")
+dual_encoder = SentenceTransformer("all-MiniLM-L6-v2")
+
+subset = "default" # subset name used in the NFCorpus dataset
+
+for task in ["NFCorpus"]: 
+    eval_splits = ["test"]
+    evaluation = MTEB(
+        tasks=[task], task_langs=["en"]
```

Comment:
Can we make sure to use the recommended way for selection? Currently raises the below warnings  cc @KennethEnevoldsen 
```
WARNING:mteb.evaluation.MTEB:The `task_langs` argument is deprecated and will be removed in the next release. Please use `tasks = mteb.get_tasks(... languages = [...])` to filter tasks instead. Note that this uses 3 letter language codes (ISO 639-3).
WARNING:mteb.evaluation.MTEB:Passing task names as strings is deprecated and will be removed in the next release. Please use `tasks = mteb.get_tasks(tasks=[...])` method to get tasks instead.

```